f8a_worker/workers/unknown_dep_fetcher.py

```diff
@@ -0,0 +1,47 @@
+from __future__ import division
+import json
+from f8a_worker.graphutils import GREMLIN_SERVER_URL_REST
+from f8a_worker.base import BaseTask
+from f8a_worker.utils import get_session_retry
+
+class UnknownDependencyFetcherTask(BaseTask):
+    _analysis_name = 'unknown_deps_fetcher'
+    description = 'Fetch unknown dependencies'
+
+    def get_dependency_data(self, dependency_list):
+        ecosystem = "maven"
```

Comment:
do we want this hard-coding?

Reply:
@miteshvp The parent GithubDependencyTreeTask implementation was done by the team assuming maven ecosystem at this moment. In order to change this, the parent implementation has to be generalized. We will implement this when we are going to add support for other ecosystems in sprint 146.